modules/ROOT/pages/release-notes.adoc

```diff
@@ -9,11 +9,23 @@ include::partial$_attributes-local.adoc[]
 :nftr: New Features
 :api: API Changes
 
+== 2.7.3 -- May, 2020
+
+*{fixed}*
+
+{jira-url}/CBG-816[*CBG-816*] -- Bump gocbconnstr dependency to uptake SRV support::
+Update Sync Gateway references to pick-up the latest version of `gocbconnstr`, to ensure connection to servers using SRV records.
+
+*{ke}*
+
+* {jira-url}/CBG-661[*CBG-661*] -- Errors from REST API produce invalid JSON
+
```

Comment:
This fix was backported to 2.7.2, so shouldn't be a known issue in 2.7.3
https://issues.couchbase.com/browse/CBG-750

Reply:
Resolved by removing known error cfb7b5cb34edd309dd787ba76bd4b4329d5a9c86